tests/std/tests/P0896R4_ranges_alg_copy_n/test.cpp

```diff
@@ -9,34 +9,27 @@
 
 #include <range_algorithm_support.hpp>
 
-constexpr void smoke_test() {
-    using ranges::copy_n, ranges::copy_n_result, ranges::iterator_t;
-    using std::same_as;
+using namespace std;
+using same_as;
+using ranges::copy_n, ranges::copy_n_result, ranges::iterator_t;

```

Comment:
`using ranges::copy_n` here won't have the desired effect since there's already a `copy_n` in this scope via the using directive. I'd move this line into `instantiator::call` and qualify the uses on line 17.

Reply:
Yep that was what I meant. These using directives are super evil